modules/caddyhttp/fileserver/matcher.go

```diff
@@ -68,6 +68,15 @@ type MatchFile struct {
 	//
 	// Default is first_exist.
 	TryPolicy string `json:"try_policy,omitempty"`
+
+	// A list of delimiters to use to split the path in two
+	// when trying files. If empty, no splitting will
+	// occur, and the path will be tried as-is. For each
+	// split value, the left-hand side of the split,
+	// including the split value, will be the path tried.
+	// For example, the path `/remote.php/dav/` using the
+	// split value `.php` would try the file `/remote.php`.
```

Comment:
Do you think the godoc should mention that all delimiters will be suffixed with `/`?

Reply:
Technically they're not all suffixed, because `/remote.php` still works (without a final `/`), but yeah it could mention it

Reply:
Well, at least mention that the delimiters have to be at the end of either the path or the filename, i.e. have to be followed by a `/` if not at the end of the path.